src/vm/dwbucketmanager.hpp

```diff
@@ -355,15 +354,6 @@ BaseBucketParamsManager::BaseBucketParamsManager(GenericModeBlock* pGenericModeB
     INDEBUG(m_countParamsLogged = 0);
     
     ZeroMemory(pGenericModeBlock, sizeof(GenericModeBlock));
-
-    EECodeInfo codeInfo(initialFaultingPc);
-    if (codeInfo.IsValid())
-    {
-        m_pFaultingMD = codeInfo.GetMethodDesc();
```

Comment:
This should stay on the `if (m_pFailingMD) FindFaultingMethodInfo();` part that adjusts `m_pFaultingMD` should be deleted.

Reply:
Readded